main.go

```diff
@@ -20,6 +21,10 @@ func main() {
 			logging.Debugf("user interrupted the process")
 		case errors.As(err, &userErr):
 			fmt.Fprintln(os.Stderr, userErr.Error())
+		case strings.Contains(err.Error(), "x509: certificate signed by unknown authority"):
+			// Cert error is most likely caused by mismatch between the client session and its cached server
+			// eg) server was re-installed, but the client cache was not cleared
+			fmt.Fprintf(os.Stderr, "Session certificate is no longer valid for this server; run 'infra login' to start a new session:\n\n%v\n", err)
```

Comment:
Maybe "TLS certificate ..." ? In a lot of installs the server TLS certificate will not change, so it won't break on each restart. 

```suggestion
			fmt.Fprintf(os.Stderr, "TLS certificate is no longer valid for this server; run 'infra login' to start a new session:\n\n%v\n", err)
```

Reply:
I'm not 100% sure of when the error pops up, but I wouldn't think that as a user that `infra login` would sort out a TLS certificate problem. I think most people would think that a TLS certificate error is an issue with the server, and not the client.

Reply:
It is, but the client will prompt them to fix in on login.

This is also only an issue with self-signed certs, so maybe something we are less concerned about now?

Reply:
```> infra users list
Error: GET "/api/users": Get "https://localhost/api/users?group=&limit=0&name=&page=1": x509: certificate signed by unknown authority (possibly because of "crypto/rsa: verification error" while trying to verify candidate authority certificate "Infra Server")

To see if the issue is with the client, run `infra login` again OR `infra logout --clear` to clear session```